src/components/App/App.js

```diff
@@ -74,24 +73,23 @@ class App extends React.Component {
         rating={this.state.currentMovie.movie.average_rating}
         goBack={this.backToMain}
       />
-    )
+    );
   }
 
   render() {
```

Comment:
I recommend keeping your `render` method as clean as possible. Consider moving the logic to another function and invoking it from within `render`